tests-configuration-infinispan/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/remote/cache/container/configuration/AttributesTest.java

```diff
@@ -119,6 +122,7 @@ public void toggleTCPNoDelay() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
What is the reason for ignore?

Reply:
UI model change, attribute that is subject of testing became deprecated. Resolving.